src/GitPrompt.ps1

```diff
@@ -101,6 +101,8 @@ $global:GitPromptSettings = [pscustomobject]@{
     DefaultPromptDebugSuffix                    = ' [DBG]$(''>'' * ($nestedPromptLevel + 1)) '
     DefaultPromptEnableTiming                   = $false
 
+    AbbreviateHomeDirectory                     = $true

```

Comment:
After re-reading, this change only affects the default prompt, so sure.

Would `DefaultPromptAbbreviateHomeDirectory` be appropriate?